solution/backend/common/functions.py

```diff
@@ -50,7 +50,15 @@ def loadSeedData():
         ("cmcs_regulations/fixtures/auth.permission.json", Permission),
         ("cmcs_regulations/fixtures/auth.group.json", Group),
     ]
+    # The vector column for the search index has issues with using seed data.  As in it takes forever.
+    # We remove the column first, then we add it back after fixtures are done.
 
+    file_index = ContentIndex.objects.all()
+    for con in file_index:
+        con.locations.clear()
+        con.subjects.clear()
+        con.save()
+    file_index._raw_delete(file_index.db)
```

Comment:
_raw_delete is faster but its very blunt.  Ignores things the many to many relationships and just essentially deletes the table.  Not something you always want to do but its a good use here.